firebase-installations-interop/src/main/java/com/google/firebase/installations/InstallationTokenResult.java

```diff
@@ -21,30 +21,44 @@
 @AutoValue
 public abstract class InstallationTokenResult {
 
-  /** A new FIS Auth-Token, created for this Firebase Installation. */
+  /**
+   * A new FIS Auth-Token, created for this Firebase Installation.
+   *
+   * @hide
```

Comment:
I think `getToken()` was part of the API-reviewed, no?

Reply:
You are right. Updated the files.